lib/fluent/plugin/out_forward/failure_detector.rb

```diff
@@ -0,0 +1,84 @@
+#
+# Fluentd
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+require 'fluent/output'
```

Comment:
`fluent/plugin/output`

Reply:
Is it okay not to do `require 'fluent/compat/output'`?
fluent/plugin/output doesn't do `require 'fluent/compat/output'`, does it?

Reply:
`fluent/output` and `fluent/compat/output` is for v0.12 API plugins.

Reply:
ah.I got it. https://github.com/fluent/fluentd/pull/2483/commits/2520e640a99ada6f0b00e1cb34cf91ca36dd44c1